portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFields.testcase

```diff
@@ -1499,4 +1813,95 @@ definition {
 		}
 	}
 
+	@description = "LPS-187522. Allow administrators to configure client extension filters on dataset views."
+	@priority = 5
+	test ValidationTestforCreationofClientExtensionFrontendDataSetFilterwithFieldinUs {
```

Comment:
Here you are testing that the user cannot create a client extension filter using a client extension already in use, so the name of the test should be something like "ClientExtensionFilterInUse"